onduty/.eslintrc.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
Thoughts on using release-calendar's eslint rules as a baseline? If you take out the React things, you get some nice stuff like 
```
    'notice/notice': [
      'error',
      {
        'mustMatch': 'Copyright 20\\d{2} The AMP HTML Authors\\.',
        'templateFile': path.resolve(__dirname, '../build-system/LICENSE-TEMPLATE.txt'),
        'messages': {
          'whenFailedToMatch': 'Missing or incorrect license header'
        }
      }
    ],
```

and 

```
    'sort-imports-es6-autofix/sort-imports-es6': [
      'error',
      {
        'ignoreCase': false,
        'ignoreMemberSort': false,
        'memberSyntaxSortOrder': ['none', 'all', 'multiple', 'single']
      }
    ],
```

Reply:
That `notice` rule is pretty nifty! I've added these two for now, and created https://github.com/ampproject/amp-github-apps/issues/809 to track/figure out how to properly standardize our rules without continuing to copy-paste configs between apps.